doc/manuals/admin/statistics.md

```diff
@@ -0,0 +1,215 @@
+counters are not used in all places, so race conditions on request running at the same time could
+(theoretically) lead to losing some data. However, under low load conditions, statistics should
+be accurate. There is [an open issue about this](https://github.com/telefonicaid/fiware-orion/issues/1504).
+
+## GET /statistics
+
+The statistics JSON is composed of four conditional blocks and two unconditional fields:
+
+```
+{
+  "counters" : { ... },
+  "semWait" : { ... },
+  "timing" : { ... },
+  "notifQueue": { ... },
+  "uptime_in_secs" : "65697",
+  "measuring_interval_in_secs" : "65697"
+}
+```
+
+Conditional blocks are enabled using `-statXXX` flags at the command line (in the future, maybe
+we will develop the posibility of activatinig/deactiving them using the [management API](management_api.md).
+This is due to two reasons. First, to avoid too verbose statistics output for users that are not interested
+in want some parts. Second, measure some of the information could involve a performance penalty
+(in general, to measure time always needs additional system calls) so explicit activation is needed.
+
+* "counters" (enabled with the `-statCounters`)
+* "semWait" (enabled with the `-statSemWait`)
+* "timing" (enabled with the `-statTiming`)
+* "notifQueue" (enabled with the `-statNotifQueue`)
+
+Unconditional fields are:
+
+* `uptime_in_secs`, Orion uptime in seconds.
+* `measuring_interval_in_secs`, statistics measuring time in seconds. It is set to 0 each time statistics
+  are reset. If statistics have not been reset since Orion start, this field matches `uptime_in_secs`.
+
+### Counter block
+
+It provides counters information for the time a particular request type has been received (also for
+notifications being sent), e.g:
+
+```
+{
+  ...
+  "counters" : {
+    "jsonRequests" : "75916",
+    "queries" : "3698",
+    "updates" : "2416",
+    "subscriptions" : "138",
+    "unsubscriptions" : "6",
+    "notificationsReceived" : "216936",
+    "notificationsSent" : "579542",
+    "individualContextEntity" : "360",
+    "allContextEntitiesRequests" : "3",
+    "versionRequests" : "1109",
+    "statisticsRequests" : "13",
+    "invalidRequests" : "2"
+  },
+  ...
+}
+```
+
+If a particular request type has not been received, its corresponding counter is not shown.
+
+### SemWait block
+
+It provides accumulated waiting time for main internal semaphores. It can be useful to detect bottelnecks, e.g.
+if your DB is two slow or your DB pool is undersized, then `dbConnectionPool` time would be anormaly high
+(see section on [performance tunning](perf_tuning.md) for more details).
+
+```
+{
+  ...
+  "semWait" : {
+    "request" : "0.000000000",
+    "dbConnectionPool" : "2.917002794",
+    "transaction" : "0.567478849",
+    "subCache" : "0.784979145",
+    "curlContext" : "0.000000000",
+    "timeStat" : "0.124000605"
+  },
+  ...
+}
+```
+
+### Timing block
```

Comment:
@kzangeli  please have a look specially to this section, in order to check if the different counter are correctly described.